tests/test_qiskit_device.py

```diff
@@ -13,6 +73,23 @@
 ]
 
 test_device_options = [{}, {"optimization_level": 3}, {"optimization_level": 1}]
+backend = MockedBackend()
+legacy_backend = MockedBackendLegacy()
```

Comment:
So I know this is a bit of a 180 from what I said previously wrt the mocking, but what does stop us from just using the `FakeManila` and `FakeManilaV1` backends? Do they require a token? I.e. just:

```
from qiskit_ibm_runtime.fake_provider import FakeManilaV2, FakeManila
```

and then we replace this with 

```
backend = FakeManillaV2()
legacy_backend = FakeManila()
```

Reply:
As far as I know, they don't require a token. I think we could just use `FakeManila` for the tests as well. I'll add it as an additional test case.